chia/farmer/farmer_api.py

```diff
@@ -58,6 +58,10 @@ class FarmerAPI:
     def __init__(self, farmer: Farmer) -> None:
         self.farmer = farmer
 
+    @property
```

Comment:
Are these used enough that we can't just make it a regular function?  I'd consider a property to be a tansitional tool when trying to retain back-compat, but it seems that we have just the single use that we're already updating.

I'm also vaguely itchy about the repetition of `api`, but not entirely sure I should be.  As in, `whatever_api.api_ready` is repetitive vs. just naming this `.ready()`.

Reply:
I was tempted to remove the `api_` prefix also but i didn't want to make this a change PR actually, just introduce a protocol about what exists. Since its a bigger change due to your PR anyway now, i added d64dbf89243ab64716fa18857a6c11a6a68c6bd5 :)